Homeworks/homework_6/airports/airport.py

```diff
@@ -0,0 +1,40 @@
+
+class Airport:
+    def __init__(self, port_info):
+        self.hangars = []
+        self.hangars_amount = int(port_info['num_hangars'])
+        self.coordinates = [float(port_info['latitude_deg']), float(port_info['longitude_deg'])]
+        self.port_id = port_info['id'] + port_info['ident']
+        self.name = port_info['name']
+        self.airport_type_description = port_info['type']
```

Comment:
it supposed to be class attribute, because  it shared feature among the class instances